src/Posts/Feed.jsx

```diff
@@ -27,26 +51,25 @@ const renderItem = (item) => {
         props={{
           accountId: item.account_id,
           blockHeight: item.block_height,
+          blockTimestamp: item.block_timestamp,
           content: item.content,
           comments: item.comments,
           likes: item.accounts_liked,
           GRAPHQL_ENDPOINT,
           verifications: item.verifications,
           showFlagAccountFeature: props.showFlagAccountFeature ?? false,
+          profile: returnProfileForUser(item),
```

Comment:
I assume passing this saves some queries within Posts.Post?

Reply:
Correct. I chatted with @gabehamilton, and his plan is to pass the profile info of each author with Query API. This will prevent us having to call `Social.get()` inside `AccountProfile` to fetch the user's display name and avatar.

Reply:
https://github.com/near/near-discovery-components/issues/427